docs/index.md

```diff
@@ -76,3 +81,9 @@ The Yeep API provides programmatic access to users, permissions, roles, sessions
 | **[totp.enroll](methods/totp.enroll.md)**     | Enrolls the designated user to TOTP authentication          |
 | **[totp.activate](methods/totp.activate.md)** | Activates TOTP authentication for the designated user       |
 | **[totp.eject](methods/totp.eject.md)**       | Removes TOTP authentication factor from the designated user |
+
+### Email
+
+| Method                                                | Description                                    |
+| :---------------------------------------------------- | :--------------------------------------------- |
+| **[email.verify](methods/invitation.accept.md)**      | Verifies user email address by redeeming verification token |
```

Comment:
I am confused as to why you need the `email.verify` method in the first place. I thought the email verification flow would work as follows....

1. User triggers email verification via API (i.e. `user.verifyEmail`)
2. User receives email message at their inbox with verification token + link. For this we need the email template + event that you already created.
3. Upon clicking on the link the user verifies the email address and gets back an HTML page. This should be the public page you created.

So why do we need the `email.verify()` API method? What am I missing?